docker/Dockerfile

```diff
@@ -1,19 +1,41 @@
+FROM groovy:jdk8 as build
+
+USER root
+RUN apt-get update \
+    && apt-get install -yq make
+#USER groovy
+
+WORKDIR /src
+COPY . /src
+
+RUN make deps
```

Comment:
if `make deps` only uses a subset of the files, it would be good to change the COPY line above to only bring in those, and then COPY the rest after `make deps`